trillian/mockclient/mock_log_client.go

```diff
@@ -69,14 +68,14 @@ func (m *MockTrillianLogClient) GetConsistencyProof(arg0 context.Context, arg1 *
 	return ret0, ret1
 }
 
-// GetConsistencyProof indicates an expected call of GetConsistencyProof.
+// GetConsistencyProof indicates an expected call of GetConsistencyProof
```

Comment:
I think you may be generating this with an older version of the mock tooling. I think they went from *not* having a terminating period to having it.